services/core-api/app/api/incidents/models/mine_incident_category.py

```diff
@@ -9,10 +9,10 @@
 
 class MineIncidentCategory(AuditMixin, Base):
     __tablename__ = 'mine_incident_category'
-    mine_incident_category_code = db.Column(db.String, primary_key=True)
+    mine_incident_category_code = db.Column(db.String(3), primary_key=True)
     description = db.Column(db.String, nullable=False)
     active_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
 
     @classmethod
     def active(cls):
-        return cls.query.filter_by(active_ind=True).all()
+        return cls.query.filter_by(active_ind=True).order_by(cls.description).all()
```

Comment:
Sorting it alphabetically will make it easier for users to find a code in the dropdown, especially as more codes are added. This would be redundant if we decide to sort all of the records in the mine_incident_category table by default as we insert new codes.

Reply:
This also appears to be inconsistent in our app, but we also have `display_order` as column to do just this. assuming it's stable even without your change, i'd rather not have a 3rd style introduced. let me know your thoughts.

Reply:
Are you suggesting I add a display_order (integer) column to the mine_incident_category table and then do order_by(cls.display_order) instead? If so, I could do that no problem.

Reply:
Yea, that would be best version of the code table we have so far.

Reply:
I will make that change.